tests/test_marathon.py

```diff
@@ -541,7 +536,7 @@ def _assert_add_pod_returns_parsed_response_body(response_json):
     marathon_client, rpc_client = _create_fixtures()
     rpc_client.http_req.return_value = mock_response
 
-    assert marathon_client.add_pod({'some': 'json'}) == response_json
+    assert marathon_client.add_pod({'some': 'json'}) startswith "Marathon-Deployment-ID"
```

Comment:
Did you try running this locally? I don't see how this will work, since you haven't set the header value for `Marathon-Deployment-Id` on the mock response object.

Since `response_json` isn't used anymore, could you rename it to `deployment_id` and use it to pass in the expected header value?